test-clients/src/main/java/com/hedera/services/bdd/suites/ethereum/EthereumSuite.java

```diff
@@ -497,6 +511,115 @@ HapiApiSpec ETX_009_callsToTokenAddresses() {
 	}
 
 
+	private HapiApiSpec ETX_008_contractCreateExecutesWithExpectedRecord() {
+		final var txn = "creation";
+		final var contract = "Fuse";
+
+		return defaultHapiSpec("ChildCreationsHaveExpectedKeysWithOmittedAdminKey")
```

Comment:
You can change the spec name to be the same like the test function name - `ETX_008_contractCreateExecutesWithExpectedRecord`